packages/backend-common/src/reading/types.ts

```diff
@@ -83,6 +83,8 @@ export type ReadTreeResponseFactoryOptions = {
   etag: string;
   // Filter passed on from the ReadTreeOptions
   filter?: (path: string, info?: { size: number }) => boolean;
+  // First directory of a tar archive is stripped when set to true
+  tarStripFirstDirectory?: boolean;
```

Comment:
I don't think there's any reason to put this here in the common options type, is there? You could just as well have made

```ts
export interface ReadTreeResponseFactory {
  fromTarArchive(
    options: ReadTreeResponseFactoryOptions & {
      stripFirstDirectory?: boolean
    },
  ): Promise<ReadTreeResponse>;
```

right?

Reply:
Yes, I forgot we could do that, looks better like this!